build.go

```diff
@@ -380,3 +396,27 @@ func contains(candidates []string, s string) bool {
 
 	return false
 }
+
+func ValidateSBOMFormats(layersPath string, acceptedSBOMFormats []string) error {
+	sbomFiles, err := filepath.Glob(fmt.Sprintf("%s/*.sbom.*", layersPath))
```

Comment:
Can we use filepath join here to combine the glob with the layer path?

Reply:
```suggestion
	sbomFiles, err := filepath.Glob(filepath.Join(layersPath, "*.sbom.*"))
```